Formula/dotnet.rb

```diff
@@ -0,0 +1,68 @@
+class Dotnet < Formula
+  desc ".Net Core"
+  homepage "https://dotnet.microsoft.com/"
+  url "https://github.com/dotnet/source-build.git",
+      tag:      "v3.1.108-SDK",
+      revision: "05a0c8f6b13fcbec4b441c977b639b75e8a74f67"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "curl"
+  depends_on "icu4c"
+  depends_on "openssl"
+
+  def install
+    # Arguments needed to not artificially time-limit downloads from Azure.
+    # See the following GitHub issue comment for details:
+    # https://github.com/dotnet/source-build/issues/1596#issuecomment-670995776
+    system "./build.sh", "/p:DownloadSourceBuildReferencePackagesTimeoutSeconds=N/A",
+                         "/p:DownloadSourceBuiltArtifactsTimeoutSeconds=N/A"
+
+    libexec.mkpath
+    doc.mkpath
+    tarball = Dir["artifacts/*/Release/dotnet-sdk-#{version}-*.tar.gz"].first
+    system "tar", "-xzf", tarball, "--directory", libexec
+    mv Dir[libexec/"*.txt"], doc
```

Comment:
```suggestion
    doc.install Dir[libexec/"*.txt"]
```

Reply:
Great, thanks! It would be awesome if something like `libexec.install Dir["artifacts/*/Release/dotnet-sdk-#{version}-*.tar.gz"], extract: true` could replace the following three lines as well:

```ruby
libexec.mkpath
tarball = Dir["artifacts/*/Release/dotnet-sdk-#{version}-*.tar.gz"].first
system "tar", "-xzf", tarball, "--directory", libexec
```